test/serialisers/json-0.6.js

```diff
@@ -185,7 +185,7 @@ describe('JSON 0.6 Serialiser', function() {
 
     it('serialises enum', function() {
       var defaultElement = new minim.Element(new minim.elements.String('North'));
-      defaultElement.element = 'default';
+      defaultElement.element = 'enum';
```

Comment:
I spotted this was incorrect while copying the test case below to make a fixed variant.